lib/player.js

```diff
@@ -6321,44 +6356,43 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
 
     let liveSyncMaxLatency;
     let liveSyncPlaybackRate;
+    let liveSyncMinLatency;
+    let liveSyncMinPlaybackRate;
+    const liveSyncTargetLatencyTolerance =
+      this.config_.streaming.liveSyncTargetLatencyTolerance;
+
     if (this.config_.streaming.liveSync) {
-      liveSyncMaxLatency = this.config_.streaming.liveSyncMaxLatency;
+      if ('liveSyncTargetLatency' in this.config_.streaming) {
```

Comment:
at this point, we probably have a target latency configured, but it felt correct to guard against it.

Reply:
This is not necessary, because our default config has it. Please remove this if.